Action/src/Action.java

```diff
@@ -0,0 +1,109 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Action {
+    public static void main(String[] args) throws IOException {
+        int countOfWater = 400;
+        int countOfMilk = 540;
+        int countOfCoffeeBeans = 120;
+        int countOfMoney = 550;
+        int countOfCups = 9;
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("The coffee machine has:");
+        System.out.println(countOfWater + " of water");
+        System.out.println(countOfMilk + " of milk");
+        System.out.println(countOfCoffeeBeans + " of coffee beans");
+        System.out.println(countOfMoney + " of money");
+        System.out.println("Write action (buy, fill, take):");
+        System.out.print(">");
+        String action = reader.readLine();
+
+        if(action.equals("buy"))
```

Comment:
@AraChilingaryan good example to use the switch case

Reply:
done